modules/report/src/main/Report.scala

```diff
@@ -53,10 +53,11 @@ object Report {
 
   case class Inquiry(mod: User.ID, seenAt: DateTime)
 
-  case class WithUser(report: Report, user: User, isOnline: Boolean) {
+  case class WithUser(report: Report, user: User, isOnline: Boolean, accuracy: Option[Int], room: Option[Room]) {
 
     def urgency: Int =
-      (nowSeconds - report.createdAt.getSeconds).toInt +
+      (if (room has Room.Cheat) accuracy getOrElse 0
```

Comment:
no I think this applies to all cheat reports, including the ones in the All room. In any case, I don't think accuracy should impact ordering.